app/models/cohort.rb

```diff
@@ -0,0 +1,15 @@
+class Cohort < ApplicationRecord
+  validates :start_year,
```

Comment:
Might also be worth adding a validation for the `registration_start_date` here so that the year is the same as the `start_year`?